docs/README.md

```diff
@@ -1 +1,21 @@
 # dtpcontrols
+
+
+## Quickstart recipe for firmware developers & testers
+
+Here is a recipe for installing dtpcontrols and uhallibs onto dunedaq-v2.9.0.
+
+```
+source /cvmfs/dunedaq.opensciencegrid.org/setup_dunedaq.sh
+setup_dbt dunedaq-v2.9.0
+dbt-create.sh workDir
+cd workDir/sourcecode
+git clone https://github.com/DUNE-DAQ/dtpcontrols.git -b python_fixes
```

Comment:
Watch out, once this PR is merged, the `python_fixes` branch will be no more.

Reply:
thanks - fixed.